website/addons/wiki/templates/edit.mako

```diff
@@ -354,42 +353,39 @@
 </div>
 
 <%def name="javascript_bottom()">
-<% import json %>
 ${parent.javascript_bottom()}
 <script>
 
     var canEditBody = ${user['can_edit_wiki_body'] | sjson, n};
     var isContributor = ${user['can_edit']  | sjson, n};
 
-    var canEditPageName = isContributor && ${json.dumps(
-        wiki_id and wiki_name != 'home'
-    )};
+    var canEditPageName = isContributor && ${(wiki_id and wiki_name != 'home') | sjson, n };
 
     window.contextVars = window.contextVars || {};
     window.contextVars.wiki = {
         canEdit: canEditBody,
         canEditPageName: canEditPageName,
-        usePythonRender: ${json.dumps(use_python_render)},
```

Comment:
Could you briefly explain why we used to need json.dumps here and we don'y any more?